Source/WebKit/UIProcess/win/WebView.cpp

```diff
@@ -544,22 +550,24 @@ LRESULT WebView::onPrintClientEvent(HWND hWnd, UINT, WPARAM wParam, LPARAM, bool
 
 LRESULT WebView::onSizeEvent(HWND hwnd, UINT, WPARAM, LPARAM lParam, bool& handled)
 {
-    int width = LOWORD(lParam);
-    int height = HIWORD(lParam);
-
-    IntSize windowSize(width, height);
+    // FIXME: default device scale factor was not used at first loading.
+    // If there are no m_page, use intrinsic device scale factor.
+    if (!m_page)
+        m_viewSize = expandedIntSize(FloatSize(LOWORD(lParam), HIWORD(lParam)) / deviceScaleFactorForWindow(hwnd));
+    else
+        m_viewSize = expandedIntSize(FloatSize(LOWORD(lParam), HIWORD(lParam)) / m_page->deviceScaleFactor());
```

Comment:
nits: We can reduce redundancy by using ternary op
```suggestion
    float deviceScaleFactor = m_page ? m_page->deviceScaleFactor() : deviceScaleFactorForWindow(hwnd);
    m_viewSize = expandedIntSize(FloatSize(LOWORD(lParam), HIWORD(lParam)) / deviceScaleFactor);
```

Reply:
My code has fewer variables than your suggested code.
So, I think that PCs can run my code faster than yours.
Does anyone have any ideas about which is good?

Reply:
Of Course this is style nits, but it seems like clang and gcc don't have any difference if we use any optimization level over 1. https://godbolt.org/z/c87YTf3na

Reply:
I fixed this like yours.
Thank-you.